lib/tests/streamlit/elements/arrow_altair_test.py

```diff
@@ -585,6 +587,36 @@ def test_chart_with_x_y_on_sliced_data(
             orig_df=df, expected_df=EXPECTED_DATAFRAME, chart_proto=proto
         )
 
+    @parameterized.expand(ST_CHART_ARGS)
+    @unittest.skipIf(
+        version.parse(alt.__version__) < version.parse("5.0.0"),
```

Comment:
@LukasMasuch is this a typo, or intentional?
Is this should be skipped if  `pandas<2.0.0`

Reply:
ahh, wait. This refers to the wrong comment below. Fixed it 👍